app/views/users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h3> Your Events </h3>
+<p>
+   <ul id = "users">
+      <% @user.created_events.each do |event|%>
+      <li>
```

Comment:
Move this logic to the user's controller in the `show` action, i.e. `@created_events = current_user.created_events` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)). It is a better place for such logic than the view.